src/panel/context/Events.tsx

```diff
@@ -62,16 +65,16 @@ export const EventsProvider: FC = ({ children }) => {
     key: []
   });
 
-  /** Set initial state from cache */
-  useEffect(() => {
-    window.chrome.devtools.inspectedWindow.eval(
-      `window.__urql__.events`,
-      (ops: UrqlEvent[]) => {
-        console.log(ops);
-        setRawEvents(ops);
-      }
-    );
-  }, []);
+  // /** Set initial state from cache */
```

Comment:
Does this happen automatically now or are we still missing this?

Reply:
We are missing this, my thinking is we should implement the panel similar to how chrome does the network panel. Once the panel is initialy opened, requests are captured.